README.md

```diff
@@ -18,97 +18,77 @@ We're building up Pixie for broad use by the end of 2020 with an initial focus o
   <summary><strong>Table of contents</strong></summary>
 
 - [Quick Start](#quick-start)
-- [Run Scripts](#run-scripts)
-- [Demos](#Demos)
+- [Demo](#demos)
 - [Contributing](#contributing)
 - [Open Source](#open-source)
 - [Under the Hood](#under-the-hood)
 - [About Us](#about-us)
+- [License](#license)
 </details>
 
 
-## Quick start
+## Quick Start
 
-1. **Create account**
+1. **Signup**: Visit our [product page](https://work.withpixie.ai/) and signup with your google account.
 
-    Visit our [product page](https://work.withpixie.ai/) and signup with your google account.
-
-2. **Install**
-
-    Copy and run this command to install the CLI in your environment.
+2. **Install CLI:** Copy and run this command to install `px` in your environment
```

Comment:
It's not clear what px is. It's guessable by the context but not super clear. Can consider rewording.